bin/music_album.rb

```diff
@@ -0,0 +1,16 @@
+require_relative 'item'
+class MusicAlbum < Item
+  attr_accessor :on_spotify, :name
+  attr_reader :publish_date
+
+  def initialize(name, on_spotify)
+    super(publish_date)
+    @name = name
+    @publish_date = publish_date
```

Comment:
@publish_date is already set by `super()` constructor, you can delete this line.